Calendar/Get-RBASummary.ps1

```diff
@@ -131,72 +133,75 @@ function OutputRBACriteria {
     Write-Host;
 }
 
-function OutputVerboseRBACriteria {
-    if ($VerbosePreference -ne $true) {
-        return
-    }
-    $VerboseRBACriteria = ""
+# RBA processing settings Verbose Output
+function RBACriteriaExtra {
+    if ($PSBoundParameters['Verbose']) {
```

Comment:
Didn't catch this before, we should just utilize `Write-Verbose` regardless if `$PSBoundParameters['Verbose']` is true or not. If it isn't true, `Write-Verbose` will not display anything. 

Does this need to be in its own function?

Reply:
fixed